src/math/vec2.js

```diff
@@ -93,6 +93,27 @@ class Vec2 {
         return this;
     }
 
+    /**
+     * @function
+     * @name Vec2#addScalar
+     * @description Adds a number to each element of a vector.
+     * @param {number} scalar - The number to add.
+     * @returns {Vec2} Self for chaining.
+     * @example
+     * var vec = new pc.Vec2(3, 4);
+     *
+     * vec.addScalar(2);
+     *
+     * // Should output [5, 6]

```

Comment:
I'd prefer the comment to strike a confident tone and say something like 'outputs [5, 6]', but I understand the existing examples all use this language.

Reply:
Done. Thanks!